<launch>
    # download
    <node name="nclt2ros" pkg="nclt2ros" type="nclt2downloader" output="screen">

        <param name="date"            value="2012-02-04"/>
        <param name="raw_data_path"   value="/home/christian/nclt2ros/raw_data" />

        <param name="lb3"             value="False" />
        <param name="sen"             value="True" />
        <param name="vel"             value="False" />
        <param name="hokuyo"          value="False" />
        <param name="gt"              value="True" />
        <param name="gt_cov"          value="True" />

    </node>
</launch>
